src/test/kotlin/com/igorwojda/integer/countdown/challenge.kt

```diff
@@ -4,7 +4,7 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun countDown(n: Int): List<Int> {
-    TODO("not implemented")
+    return List(n+1) { i -> i }.reversed()
```

Comment:
Check out ranges (https://kotlinlang.org/docs/ranges.html) and collectors/copying (https://kotlinlang.org/docs/reference/collection-aggregate.html